tailwind.config.js

```diff
@@ -1,11 +1,55 @@
 /** @type {import('tailwindcss').Config} */
+const defaultTheme = require('tailwindcss/defaultTheme');
+
 module.exports = {
   content: [
-    "./pages/**/*.{js,ts,jsx,tsx}",
-    "./components/**/*.{js,ts,jsx,tsx}",
+    './pages/**/*.{js,ts,jsx,tsx}',
+    './components/**/*.{js,ts,jsx,tsx}',
   ],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        sans: ['var(--font-poppins)', ...defaultTheme.fontFamily.sans],
+      },
+      colors: {
+        white85: 'rgba(255, 255, 255, 0.85);',
```

Comment:
Define another opacity instead of the color white with 85% opacity is better here. So that you can use the opacity with all the colors later with tailwind.